extensions/sql-migration/src/dialog/migrationCutover/migrationCutoverDialog.ts

```diff
@@ -555,6 +608,15 @@ export class MigrationCutoverDialog {
 				}
 			});
 
+			if(tableData.length === 0)
```

Comment:
Can you confirm which conditions apply the empty table message needs to be displayed:
I believe these conditions apply:
* row count is 0
* when backups are accessed from a file share
* "SQL MI" is the migration target

I'm not sure if these apply
* when "SQL VM" is the migration target
* when backups are accessed from a blob container

Reply:
I will look into all these conditions.